### DISCUSSION TITLE
Please help me with contributions to my color list.

### DISCUSSION THREAD

**Discussion by ryojiagatsuma1004** [NAIST-SE/AttractingContributors#1135] [2024 Attraction Contributions]
I need help with my color list.

Come to my fork:
https://github.com/ryojiagatsuma1004/AttractingContributors-Color

Add to my README file.

I look forward to your contributions.

**Comment by Tz-violet** [comment]
hi, I just add color to your list. 
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Reply by Tz-violet** [reply]
thank you so much!

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project 
https://github.com/polarbearpppp/AttractingContributors-Games

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
